src/modules/intelligence/application/use_cases/translate_article.py

```
@@ -0,0 +1,245 @@
+            logger.error("tag_translation_llm_failed", language=language)
+            return {"total": len(tags), "success": 0, "failed": len(tags)}
+
+        translated_lines = [line.strip() for line in translated_text.strip().split("\n") if line.strip()]
+
+        success, failed = 0, 0
+        for i, tag_id in enumerate(tag_ids):
+            if i < len(translated_lines):
+                try:
+                    self._tag_repo.save_tag_translation(tag_id, language, translated_lines[i])
+                    success += 1
+                except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)